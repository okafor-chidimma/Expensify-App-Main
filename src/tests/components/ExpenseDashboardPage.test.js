import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashBoardPage from '../../components/ExpenseDashboardPage';

describe('Rendering <ExpenseDashBoardPage/>', () => {
  it('Renders <ExpenseDashBoardPage/> correctly', () => {
    const wrapper = shallow(<ExpenseDashBoardPage/>);
    expect(wrapper).toMatchSnapshot();
  })
})