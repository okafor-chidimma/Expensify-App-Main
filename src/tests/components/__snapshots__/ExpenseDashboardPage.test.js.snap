// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering <ExpenseDashBoardPage/> Renders <ExpenseDashBoardPage/> correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
